lib/coach/request_serializer.rb

```diff
@@ -45,10 +45,9 @@ def serialize
 
     private
 
-    # rubocop:disable Lint/RescueWithoutErrorClass
     def request_path
       @request.fullpath
-    rescue
+    rescue StandardError
```

Comment:
I... actually kind of like this autocorrect! It makes it very explicit what the default Ruby behaviour is.